lib/src/protocol_instance.rs

```diff
@@ -40,20 +41,36 @@ impl ProtocolInstance {
                 input.taiko.tx_blob_hash.unwrap()
             } else {
                 println!("kzg check enabled!");
-                let mut data = Vec::from(KZG_TRUST_SETUP_DATA);
-                let kzg_settings = KzgSettings::from_u8_slice(&mut data);
+                let data_size = KZG_TRUST_SETUP_DATA.len();
+                let aligned_data_size = data_size + (4 - (data_size % 4));
```

Comment:
```suggestion
                let aligned_data_size = (data_size + 3) / 4 * 4;
```

Doesn't really matter, but this method seems more standard and also doesn't increase the size needlessly by 4 when `data_size % 4 == 0`.